src/services/defaultClasses.js

```diff
@@ -16,6 +16,11 @@ export default {
     },
     selected: {
         'vue-ads-bg-teal-100': true,
+        'vue-ads-bg-gray-100': false,
+    },
+    exactMatch: {
+        'vue-ads-bg-yellow-300': true,
+        'vue-ads-cell-exact-match': true,
```

Comment:
The `vue-ads-cell-exact-match` class is not defined, so has no effect. Please remove it.

Reply:
The goal of this custom class is to have possibility to change styles for exact match rows outside of library. For example in your library you have default styles and exact match rows have yellow background but what if you want to change background but don't want touch source code of library? In this case this class can be useful. What do you think about it?

Reply:
@pavelav done